backend/src/models/v2/Model.ts

```diff
@@ -49,7 +49,7 @@ const ModelSchema = new Schema<ModelInterface>(
   {
     id: { type: String, required: true, unique: true, index: true },
 
-    name: { type: String, required: true },
+    name: { type: String, text: true, required: true },
```

Comment:
You likely want to run this:

```javascript
ModelSchema.index({ '$**': 'text' })
```

Which will index all fields?  We'll also want to index the model card fields.